datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -162,12 +173,30 @@ func (f *ChainDataFetcher) resLoop() {
 				logger.Error("db insertion is failed", "blockNumber", res.blockNumber, "reqType", res.reqType, "err", res.err)
 				// TODO-ChainDataFetcher add retry logic when data insertion is failed
 			} else {
-				f.updateCheckpoint(res.blockNumber.Uint64())
+				if err := f.updateCheckpoint(res.blockNumber.Int64()); err != nil {
+					logger.Error("Failed to update checkpoint", "checkpoint", res.blockNumber.Int64())
```

Comment:
Please add `err` to the log.

Reply:
Added error log on ae997fa575ef5e0a1523f54e92093ad22476092f.